assignments/closure.js

```diff
@@ -1,6 +1,10 @@
 // ==== Challenge 1: Write your own closure ====
 // Write a simple closure of your own creation.  Keep it simple!
+var attacklvl = 10;
+var defenselvl = 10;
+var totalxp = 500;
 
+function maxdamage() 
 
```

Comment:
I think something is missing from here. This is an example of a closure:

This is not actually a closure, because we are just using a variable in the global scope. The idea of a closure is that scopes continue to be available to functions defined inside of them, even if that scope seems to be otherwise discarded. For example,
```
function coolest(name) {
  return function() {
    console.log('The coolest is' + name);
  }
}
const dak = coolest('dak');
```
When we call `dak()` it will still use the `name` value property because it is stored within that functions closure, even though that scope is no longer available.